hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java

```diff
@@ -138,50 +150,51 @@ public String convertToLetter(int size) {
             IOUtil.closeResource(is);
         }
 
+        ExecutorService executor = Executors.newFixedThreadPool(1);
```

Comment:
Can we lazy init this one, if the mc.isEnabled and not otherwise?

Reply:
On a second thought, I think this whole logic is better suited inside `addManCenterInfo`. WDYT?

Reply:
@tkountis 
you are right about lazy init.I will update PR 👍 

Here is the asynchronous logic I have planned:
1. Send a request to ManCenter.
2. Add other parameters (will be processed immediately, current thread won't block).
3. Get the response from ManCenter(here the current thread will block until response is returned).

If I move whole logic into `addManCenterInfo `, code work as a synchronous(not completely). WDYT?

Reply:
Well I thought that, but then again, the other work we do while the request is processed is very just a couple of ops on a map, don't think it worths the effort of offloading the request to another thread. Just my two cents of course.

Reply:
yes, you are right. @mesutcelik WDYT?

Reply:
I agree with @tkountis